app/models/manageiq/providers/kubevirt/infra_manager/provision/cloning.rb

```diff
@@ -48,30 +50,28 @@ def start_clone(options)
       template = connection.template(source.name)
     end
 
-    # Create the representation of the new offline virtual machine, copying the spec from the template:
-    offline_vm = {
-      :metadata => {
-        :name      => name,
-        :namespace => template.metadata.namespace
-      },
-      :spec     => template.spec.to_h
-    }
+    # Load example file due to current limitations of kubevirt
+    # TODO: remove when kubevirt provides the implementation
```

Comment:
@fabiand please take a look at template processing code and let me know if there is anything we need to do differently